standards/code-style.adoc

```diff
@@ -717,56 +728,49 @@ When multiple methods are chained in the same line - for example when using
 Builders - every call to a method should go in its own line, breaking the line
 before the `.`
 
-[source, java]
+[source, kotlin]
 ----
-Picasso.with(context).load("https://i.imgur.com/kG0deA5.jpg").into(imageView);
+Picasso.with(context).load("https://i.imgur.com/kG0deA5.jpg").into(imageView)
 ----
 
-[source, java]
+[source, kotlin]
 ----
 Picasso.with(context)
         .load("https://i.imgur.com/kG0deA5.jpg")
-        .into(imageView);
+        .into(imageView)
 ----
 
 __Long parameters case__
 
 When a method has many parameters or its parameters are very long, we should
 break the line after every comma `,`
 
-[source, java]
+[source, kotlin]
 ----
-loadPicture(context, "https://i.imgur.com/kG0deA5.jpg", imageViewProfilePicture, clickListener, "Title of the picture");
+loadPicture(context, "https://i.imgur.com/kG0deA5.jpg", imageViewProfilePicture, clickListener, "Title of the picture")
```

Comment:
I realized that ```clickListener``` is likely a callback/lambda so it should be the last parameter according to the standards above.  Parameter ordering is not the scope for this example so I don't think it necessarily needs to change.

Reply:
Good eye; let's make that change so the examples don't violate other guidelines.

Reply:
Updated